lib/Output.ml

```diff
@@ -1,8 +1,8 @@
 open Base
 open Stdio
 
-let output_ppm (width:int) (height:int) =
-  let _ = printf "P3\n%d %d\n 255\n" width height in
+let ppm_content (width:int) (height:int) (file) =
```

Comment:
Wouldn't it be better if file had a type signature?

Reply:
Also, the function name could be more explicit in regards to what the function does.

Reply:
I added a type for file and changed the ppm_content function to build_image.